functions/src/notifications/populateBillHistoryNotificationEvents.ts

```diff
@@ -70,7 +82,7 @@ export const populateBillHistoryNotificationEvents = functions.firestore
       const notificationEventId = notificationEventSnapshot.docs[0].id
 
       if (historyChanged) {
```

Comment:
nit: Now that we have the guard clause on `historyChanged` on line 54, do we still need this check?

Reply:
I guess it was moreso of checking if we're updating an existing notification event vs creating